docker-compose.yml

```diff
@@ -47,13 +47,14 @@ services:
           nocopy: true
 
   db:
-    image: postgres:11.5
+    image: postgres:latest
     hostname: db
     restart: always
     environment:
       - POSTGRES_HOST_AUTH_METHOD=trust
     volumes:
-      - ${POSTGRES_DATA_DIR:-./postgres-data}:/var/lib/postgresql/data
```

Comment:
This is breaking change, all existing data will be lost. Don't do that please.

Reply:
Valid objection, however I do not see any other solution supporting windows. Any ideas how to move forward?